debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java

```diff
@@ -71,11 +73,26 @@ public static void startContainers() {
     public void canRegisterConnector() throws Exception {
         debeziumContainer.registerConnector("my-connector-1", getConfiguration(1));
 
-        String status = executeHttpRequest(debeziumContainer.getConnectorStatus("my-connector-1"));
-
-        assertThat(JsonPath.<String> read(status, "$.name")).isEqualTo("my-connector-1");
-        assertThat(JsonPath.<String> read(status, "$.connector.state")).isEqualTo("RUNNING");
-        assertThat(JsonPath.<String> read(status, "$.tasks[0].state")).isEqualTo("RUNNING");
+        // task initialization happens asynchronously, so we might have to retry until the task is RUNNING
+        Awaitility.await()
+                .pollInterval(Duration.ofMillis(250))
+                .atMost(Duration.ofSeconds(5))
+                .until(
+                        () -> {
```

Comment:
@gunnarmorling I recommend to use `org.awaitility.core.ConditionFactory.ignoreException(Class<? extends Throwable>)`

Reply:
Good idea, updated.

Reply:
Or use the correct wait method (see my other comment)  ;)

Reply:
Nice one, didn't know that. Thanks!